app-backup/restic/restic-0.9.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1 golang-vcs-snapshot
+
+DESCRIPTION="A backup program that is fast, efficient and secure"
+HOMEPAGE="https://restic.github.io/"
+SRC_URI="https://github.com/restic/restic/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+EGO_PN="github.com/restic/restic"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="test"
+
+DEPEND="test? ( sys-fs/fuse:0 )"
```

Comment:
why fuse twice? If it is runtime, pull it unconditionally

Reply:
In theory fuse was not needed at build time except for the tests, but was needed unconditionally at runtime. Maybe I understood it wrong and runtime dependencies for tests should be on RDEPEND instead of putting all tests dependencies (runtime or build time) on DEPEND?